specification/cost-management/resource-manager/Microsoft.CostManagement/stable/2022-10-01/costmanagement.benefitutilization.json

```diff
@@ -0,0 +1,504 @@
+    "BenefitUtilizationSummaryKind": {
+      "type": "string",
+      "description": "Kind of the benefit utilization summary.",
+      "enum": [
+        "IncludedQuantity",
+        "Reservation",
+        "SavingsPlan"
+      ],
+      "x-ms-enum": {
+        "name": "benefitUtilizationSummaryKind",
+        "modelAsString": true,
+        "values": [
+          {
+            "value": "IncludedQuantity",
+            "description": "Benefit utilization summary is for an IncludedQuantity resource like Databricks.",
+            "name": "IncludedQuantity"
+          },
+          {
+            "value": "Reservation",
+            "description": "Benefit utilization summary is for a Reservation that is not IncludedQuantity.",
+            "name": "Reservation"
+          },
+          {
+            "value": "SavingsPlan",
+            "description": "Benefit utilization summary is for a SavingsPlan that is not IncludedQuantity.",
+            "name": "SavingsPlan"
+          }
+        ]
+      }
+    },
+    "BenefitUtilizationSummaryProperties": {
+      "description": "The properties of a benefit utilization summary.",
+      "type": "object",
+      "properties": {
+        "armSkuName": {
+          "description": "ARM SKU name ex: Standard_D1_V2 for Reservations and ComputeSavingsPlan for SavingsPlan.",
+          "type": "string",
+          "readOnly": true
+        },
+        "benefitId": {
+          "description": "The benefit Id is the identifier of a benefit within a benefit order.",
+          "type": "string",
+          "readOnly": true
+        },
+        "benefitOrderId": {
+          "description": "The benefit order Id is the identifier for a benefit purchase. Each benefit order Id represents a single purchase transaction. For example: A reservation order contains reservations. The reservation order specifies the VM size and region for the reservations.",
+          "type": "string",
+          "readOnly": true
+        },
+        "benefitType": {
+          "description": "The benefit type ex: reservation, savings plan",
+          "type": "string",
+          "readOnly": true
+        },
+        "usageDate": {
+          "description": "Date corresponding to the utilization record. If the grain of data is monthly, it will be first day of month.",
+          "type": "string",
+          "format": "date-time",
+          "readOnly": true
+        }
+      }
+    },
+    "IncludedQuantityUtilizationSummary": {
+      "description": "Included Quantity utilization summary resource.",
+      "x-ms-discriminator-value": "IncludedQuantity",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/BenefitUtilizationSummary"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/IncludedQuantityUtilizationSummaryProperties",
+          "title": "Benefit utilization Summary for benefit kind Included Quantity."
```

Comment:
>n Summary for benefit kind Included Quantity. [](http://example.com/codeflow?start=38&length=45)

improper capitalization, please review corporate style guide as to overall style of documentation

Reply:
corrected.